### ISSUE TITLE
When building a custom model using keras.Model, a warning was triggered

### ISSUE THREAD

**Issue by TCBocean** [tensorflow/tensorflow#34676]
My environment:
window10
CUDA10.0
tensorflow-gpu 1.14

When I built a more complex model today, I triggered the following warning:

> W1128 15: 22: 59.781456 32404 ag_logging.py:145] Entity <bound method DiMP.call of <__ main __. DiMP object at 0x0000024606A21208 >> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY = 10`) and attach the full output. Cause: (unicode error) 'utf-8' codec can't decode byte 0xca in position 250: invalid continuation byte (tmp56zh3z7b.py, line 15)

I don't know if this warning will affect the model operation, and the content of the warning says that it needs to be reported. So I came here.
Because there are a lot of engineering components, I can't determine which part triggered this warning. I won't upload all the code for the time being. If I need to upload code, I will not refuse.
Here is the entire print:

> C:\software\Anaconda3\envs\CV_env\python.exe C:/Users/stars_ocean/Desktop/DiMP_TF/models/DiMP.py
WARNING: Logging before flag parsing goes to stderr.
W1128 15:22:57.572051 32404 deprecation_wrapper.py:119] From C:\Users\stars_ocean\Desktop\DiMP_TF\models\backbone\MobileNet_V3_By_Model.py:4: The name tf.enable_eager_execution is deprecated. Please use tf.compat.v1.enable_eager_execution instead.

> used backbone layers: [5, 12]
> W1128 15:22:57.689736 32404 deprecation_wrapper.py:119] From C:\Users\stars_ocean\Desktop\DiMP_TF\models\classifiter\Initializer.py:24: The name tf.keras.initializers.RandomNormal is deprecated. Please use tf.compat.v1.keras.initializers.RandomNormal instead.
> 
> W1128 15:22:57.691731 32404 deprecation_wrapper.py:119] From C:\Users\stars_ocean\Desktop\DiMP_TF\models\classifiter\Optimizer.py:36: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.
> 
> 2019-11-28 15:22:57.692470: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library nvcuda.dll
> 2019-11-28 15:22:58.922915: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties: 
> name: GeForce GTX 1050 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.62
> pciBusID: 0000:01:00.0
> 2019-11-28 15:22:58.923123: I tensorflow/stream_executor/platform/default/dlopen_checker_stub.cc:25] GPU libraries are statically linked, skip dlopen check.
> 2019-11-28 15:22:58.926473: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0
> 2019-11-28 15:22:58.926787: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
> 2019-11-28 15:22:58.930528: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties: 
> name: GeForce GTX 1050 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.62
> pciBusID: 0000:01:00.0
> 2019-11-28 15:22:58.930820: I tensorflow/stream_executor/platform/default/dlopen_checker_stub.cc:25] GPU libraries are statically linked, skip dlopen check.
> 2019-11-28 15:22:58.934215: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0
> 2019-11-28 15:22:59.447520: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1181] Device interconnect StreamExecutor with strength 1 edge matrix:
> 2019-11-28 15:22:59.447669: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1187]      0 
> 2019-11-28 15:22:59.447755: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 0:   N 
> 2019-11-28 15:22:59.453515: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 2995 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1050 Ti, pci bus id: 0000:01:00.0, compute capability: 6.1)
> W1128 15:22:59.781456 32404 ag_logging.py:145] Entity <bound method DiMP.call of <__main__.DiMP object at 0x0000024606A21208>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: (unicode error) 'utf-8' codec can't decode byte 0xca in position 250: invalid continuation byte (tmp56zh3z7b.py, line 15)
> W1128 15:23:05.219440 32404 deprecation.py:323] From C:\software\Anaconda3\envs\CV_env\lib\site-packages\tensorflow\python\autograph\impl\api.py:255: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
> Instructions for updating:
> Use tf.where in 2.0, which has the same broadcast rule as np.where
> 2019-11-28 15:23:06.199208: W tensorflow/core/common_runtime/eager/context.cc:371] Added two functions with the same name: __inference_conv_and_transpose_10931
> 
> Process finished with exit code 0
>

**Comment by TCBocean** [comment]
I know why that warning is triggered. Because i am in function
`def call (self, inputs, training = None, mask = None):`
This is caused by a long paragraph with """(omitted)""".
I deleted those comments and they didn't trigger the warning.
I think this warning is very unreasonable. I think adding a function description will not cause errors in the code, so it should not trigger any warning.
